xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp

```diff
@@ -110,14 +112,13 @@ void CGUIDialogPVRTimerSettings::SetTimer(CFileItem *item)
   m_timerType     = m_timerInfoTag->GetTimerType();
   m_bIsRadio      = m_timerInfoTag->m_bIsRadio;
   m_bIsNewTimer   = m_timerInfoTag->m_state == PVR_TIMER_STATE_NEW;
-  m_bStartAnytime = m_bIsNewTimer || m_timerInfoTag->IsStartAtAnyTime();
-  m_bEndAnytime   = m_bIsNewTimer || m_timerInfoTag->IsEndAtAnyTime();
   m_bTimerActive  = m_bIsNewTimer || m_timerInfoTag->IsActive();
+  m_bStartAnyTime = m_timerInfoTag->m_bStartAnyTime;
+  m_bEndAnyTime   = m_timerInfoTag->m_bEndAnyTime;
```

Comment:
I can see your point here. For some reason I couldn't get kodi to default the start/end times to a now() on creation if there wasn't an associated EPG tag when this was set, so I cheated and changed the default. Probably just fat fingers - I'll try again.